Shared/toast.php

```diff
@@ -102,6 +102,40 @@ function toast(type, text, duration, arguments = null) {
         // Makes the toast visible ("show" is a class that makes the toast visible through css)
         toastDiv.classList.add('show', 'toast');
 
+        function truncateOverflow(toastText) {
+            const maxChars = 70; // Set the maximum number of characters before truncating
+            const toastContent = toastText.textContent.trim();
+            if (toastContent.length > maxChars) {
+                // Truncate the content and add "..."
+                toastText.textContent = toastContent.slice(0, maxChars) + '...';
+                toastText.dataset.fullText = toastContent; // Store the full text in a data attribute
+            }
+        }
```

Comment:
Do not define and declare a function inside another function. Instead, move the truncateOverflow()-function outside the toast()-function.